### PR TITLE
Fix vint errors

### LINKED ISSUE #111: Run vint on CI
Current autoload/vimlparser.vim has lots of warnings with vint.
Fix them all and run vint on CI.

### PR DESCRIPTION
**Pull request by tyru**
Fix #111

### CODE REVIEW

**Comment by tyru** [comment]
travis checks are not showed... wtf?
image
but build seems working.
https://travis-ci.org/vim-jp/vim-vimlparser/builds/566529436

@blueyed do you know something about this...? ;)

**Comment by tyru** [comment]
@blueyed and can you review this?

File: autoload/vimlparser.vim [hunk 1]
```diff
@@ -28,7 +28,7 @@ function! vimlparser#test(input, ...)
   endtry
 endfunction
 
-function! s:numtoname(num)
+function! s:numtoname(num) abort
   let sig = printf("function('%s')", a:num)
   for k in keys(s:)
     if type(s:[k]) == type({})
```
Comment (tyru) [inline autoload/vimlparser.vim]: why vint does not show error about this? 🤔

**Comment by tyru** [comment]
image
now I figured out organization grant was revoked... (why?)
granted permission and now travis appears, but codecov disappered!? 🤣
image

**Comment by tyru** [comment]
Fixed `==` to `==#`. b0018dcf5e92ad8e0f1b428dddcdf556c16451c9
I don't know why vint did not show the errors about this though.

**Comment by tyru** [comment]
@blueyed review please 🙏

**Comment by blueyed** [comment]
Trying a different approach in https://github.com/vim-jp/vim-vimlparser/pull/139.

**Comment by blueyed** [comment]
I think #139 is slightly better by using "checkvim".
It uses the master branch to fix installation issues - I should really find some time to fix vint / release 0.4..

**Comment by blueyed** [comment]
We can pick your fixed into it - or the other way around.

**Pull request merged**